src/Sulu/Bundle/WebsiteBundle/Command/SitemapGeneratorCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Command;
+
+use Sulu\Bundle\WebsiteBundle\Sitemap\SitemapGeneratorInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SitemapGeneratorCommand extends ContainerAwareCommand
+{
+
+    protected function configure()
+    {
+        $this->setName('sulu:website:sitemap:generate')
+            ->addArgument('webspace')
+            ->setDescription('Generate Sitemap!');
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $time = microtime(true);
+        $webspaceKey = $input->getArgument('webspace');
+        $style = new OutputFormatterStyle('red');
+        $output->getFormatter()->setStyle('fire', $style);
+        $style2 = new OutputFormatterStyle('green');
+        $output->getFormatter()->setStyle('done', $style2);
+
+        $output->writeln('Start generating sitemap for: "' . $webspaceKey . '".');
+        $output->writeln('Process can take some seconds');
+
+        if (empty($webspaceKey)) {
+            $output->writeln('<fire>Error: WebspaceKey needed! e.g: "sulu:website:sitemap:generate sulu_io"</fire>');
```

Comment:
Should be simply `<error>`. The default styles are `<error>`, `<info>` and `<comment>` I would remove the custom stylings as they are not used in any other commands, and if we do use them they should be defined globally.